src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+import java.util.Scanner;
+
 public class Duke {
+    // Member attributes
+    private static int inputCount = 0;
+    private static final int MAX_COUNT = 100;
+    private static final Task[] tasks = new Task[MAX_COUNT];
+
+    // Error Messages
+    public static final String ERROR_MISSING_TASK_NO =
+            "Please specify a task number to mark.";
+    public static final String ERROR_EMPTY_INPUT =
+            "Please enter either a task to do, mark/unmark <task number> or list";
+    public static final String ERROR_MISSING_EVENT_DELIMITER =
+            "Please specify a time range for the event with the delimiter \"/at\"\n" +
+                    "SYNTAX: event <event name> /at <time range>";
+    public static final String ERROR_MISSING_DEADLINE_DELIMITER =
+            "Please specify a deadline for the task with the delimiter \"/by\"\n" +
+                    "SYNTAX: deadline <deadline name> /by <due date>";
+    public static final String ERROR_INVALID_TASK_NUMBER =
+            "The input task number you have entered is invalid.";
+    public static final String ERROR_EMPTY_LIST =
+            "The task list is empty!";
+    public static final String ERROR_INPUT_LIMIT_REACHED =
+            "We have reached the limit of " + MAX_COUNT + " items that can be added to the list.\n";
+
+    // REGEX PATTERNS
+    public static final String REGEX_PATTERN_AT = "\\s/at\\s";
+    public static final String REGEX_PATTERN_BY = "\\s/by\\s";
+    public static final String REGEX_PATTERN_WHITESPACES = "\\s";
+
```

Comment:
I like how you grouped related variables together and provided a line break for readability